kafka/src/main/java/org/axonframework/extensions/kafka/eventhandling/producer/TopicResolver.java

```diff
@@ -0,0 +1,14 @@
+package org.axonframework.extensions.kafka.eventhandling.producer;
+
+import org.axonframework.eventhandling.EventMessage;
+
+import java.util.Optional;
+import java.util.function.Function;
+
+/**
+ * Interface to determine if a message should be published to Kafka, and if so to which topic. If the result from the
+ * call is {@code Optional.empty()} is will not be published, else the result will be used for the topic.
+ */
```

Comment:
I'm missing an `@author` and `@since` tag.
Now, the author isn't a hard requirement IMO, but I do think you deserve the credits here. :-)